bin/cylc-get-host-metrics

```diff
@@ -154,19 +137,41 @@ def process_load():
     String keys e.g. 'load:1' give minutes average was measured over.
     """
     cmd, av_times, patt = LOAD_TEMPLATES
-    av_raw_tuple = extract_pattern(patt, run_command_string(cmd))
+    av_raw_tuple = extract_pattern(patt, run_command(cmd))
     av_floats_tuple = [float(av_raw) for av_raw in av_raw_tuple]
     return dict(zip(
         ["load:" + str(time) for time in av_times], av_floats_tuple))
 
 
 def process_memory():
-    """Return dict with integer usable memory value from '/proc/meminfo'."""
-    cmd, patt = MEMORY_TEMPLATES
-    mem_tuple = extract_pattern(patt, run_command_string(cmd))
-    # Sum free, buffer and cache memory values (free plus/minus buffer/cache).
-    usable_memory = sum([int(mem_val) for mem_val in mem_tuple])
-    return {"memory": usable_memory}
+    """Return available memory.
+
+    Note the `free` command is inconsistent accross Linux distros.
+
+    Extracts the `MemAvailable` field from `/proc/meminfo` if present else
+    returns the sum of `MemFree`, `Buffers` and `Cached`.
+
+    ('free -k' command inconsistent across Linux distros, see e.g.
+     'https://access.redhat.com/solutions/406773', so check '/proc' directly)
+
+    """
+    stdout = run_command(['cat', '/proc/meminfo'])
```

Comment:
Just read the file with Python `open`?

Reply:
Or even run `grep MemAvailable /proc/meminfo`?

Reply:
Yes, we have refrained from using `psutil` in the pass because it would add a dependency. We don't need to worry about that any more! 😄

Reply:
Would need to do a deco to check if `psutil` has the required bells and whistles but that would be a much nicer solution. Quite happy to go down that route.

Reply:
(See #3391)